libsplinter/src/node_id/store/diesel/operations/mod.rs

```diff
@@ -12,7 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Provides NodeIdStore Operations to NodeIdStore implementors
+//! Provides [NodeIdStore](../../trait.NodeIdStore.html) Operations to
```

Comment:
This tightly couples our documentation to whatever html files the documentation renderer produces, which changes from time to time. I guess there was a 2017 [intra-rustdoc-links](https://rust-lang.github.io/rfcs/1946-intra-rustdoc-links.html) update that created a more reliable syntax.

So here I believe a better syntax would be `[NodeIdStore](super::NodeIdStore)`.